dashboard/src/components/partials/EntitiesList/EntitiesListItem.js

```diff
@@ -36,9 +42,36 @@ class EntitiesListItem extends React.PureComponent {
     `,
   };
 
+  _renderMenu = renderProps => {
+    const { open, onClose, anchorEl } = renderProps;
+    const { onClickDelete } = this.props;
+
+    return (
+      <Menu keepMounted open={open} onClose={onClose} anchorEl={anchorEl}>
+        <ConfirmDelete key="delete" onSubmit={ev => onClickDelete(ev)}>
```

Comment:
unnecessary function wrapping, do you prefer this over `onSubmit={onClickDelete}`?